Package.swift

```diff
@@ -15,19 +15,34 @@ let package = Package(
     dependencies: [
         .package(url: "https://github.com/apple/swift-nio.git", from: "2.62.0"),
         .package(url: "https://github.com/vapor/sqlite-nio.git", from: "1.8.4"),
-        .package(url: "https://github.com/vapor/sql-kit.git", from: "3.28.0"),
+        .package(url: "https://github.com/vapor/sql-kit.git", from: "3.29.3"),
         .package(url: "https://github.com/vapor/async-kit.git", from: "1.19.0"),
     ],
     targets: [
-        .target(name: "SQLiteKit", dependencies: [
-            .product(name: "NIOFoundationCompat", package: "swift-nio"),
-            .product(name: "AsyncKit", package: "async-kit"),
-            .product(name: "SQLiteNIO", package: "sqlite-nio"),
-            .product(name: "SQLKit", package: "sql-kit"),
-        ]),
-        .testTarget(name: "SQLiteKitTests", dependencies: [
-            .product(name: "SQLKitBenchmark", package: "sql-kit"),
-            .target(name: "SQLiteKit"),
-        ]),
+        .target(
+            name: "SQLiteKit",
+            dependencies: [
+                .product(name: "NIOFoundationCompat", package: "swift-nio"),
+                .product(name: "AsyncKit", package: "async-kit"),
+                .product(name: "SQLiteNIO", package: "sqlite-nio"),
+                .product(name: "SQLKit", package: "sql-kit"),
+            ],
+            swiftSettings: swiftSettings
+        ),
+        .testTarget(
+            name: "SQLiteKitTests",
+            dependencies: [
+                .product(name: "SQLKitBenchmark", package: "sql-kit"),
+                .target(name: "SQLiteKit"),
+            ],
+            swiftSettings: swiftSettings
+        ),
     ]
 )
+
+var swiftSettings: [SwiftSetting] { [
+    .enableUpcomingFeature("ConciseMagicFile"),
+    .enableUpcomingFeature("ForwardTrailingClosures"),
+    .enableUpcomingFeature("DisableOutwardActorInference"),
+    .enableExperimentalFeature("StrictConcurrency=complete"),
```

Comment:
This won't work in the 5.8 manifest

Reply:
... Now he tells me 😂

Reply:
In the past I've been leaving these in the older manifest because they're no-ops, but in retrospect it really makes more sense not to pretend that features are there when they aren't.